@xen-orchestra/proxy/src/app/mixins/backups/_RemoteAdapter.js

```diff
@@ -105,15 +106,59 @@ export class RemoteAdapter {
     }
   }
 
-  async _findPartition(diskId, partitionId) {
-    const partitions = await this.listPartitions(diskId)
+  async _findPartition(devicePath, partitionId) {
+    const partitions = await listPartitions(devicePath)
     const partition = partitions.find(_ => _.id === partitionId)
     if (partition === undefined) {
       throw new Error(`partition ${partitionId} not found`)
     }
     return partition
   }
 
+  @decorateResult(getDebouncedResource)
+  @decorateWith(deduped, (devicePath, partition) => [devicePath, partition.id])
+  @decorateWith(disposable)
+  async *_getLvmPhysicalVolume(devicePath, partition) {
+    const path = (
+      await fromCallback(execFile, 'losetup', [
+        '-o',
+        partition.start * 512,
+        '--sizelimit',
+        partition.size,
```

Comment:
We probably need to update the documentation.

Reply:
It cannot be done in this PR because it should be merged in the master branch. I noted this info, i will do it in an other PR.